internal/pkg/shared/utils.go

```diff
@@ -37,6 +48,111 @@ func getEnvVars(defaults map[string]string, vars []corev1.EnvVar) []corev1.EnvVa
 	return allVars
 }
 
+func GenerateKeystore(alias string, pw []byte) []byte {
```

Comment:
Same here: please use "password", or if you can't, passwd or similar that is more clear and less ambiguous.